concent_api/core/message_handlers.py

```diff
@@ -954,7 +954,7 @@ def store_subtask(
         raise CreateModelIntegrityError
 
 
-def handle_messages_from_database(client_public_key: bytes) -> Union[message.Message, None]:
+def handle_messages_from_database(client_public_key: bytes) -> Union[HttpResponse, message.Message, None]:
```

Comment:
`handle_messages_from_database` returns `HttpResponse`? I think it should be done by raising exception and it should be caught in decorators.